opm/parser/eclipse/EclipseState/Schedule/DynamicState.hpp

```diff
@@ -68,6 +68,17 @@ namespace Opm {
         }
 
 
+        const T& back() const {
+            if (m_data.size() > 0)
+                return m_data.back();
+            else
+                return m_initialValue;
+        }
+
+        T& back() {
```

Comment:
It's actually not needed anymore at all, I think it can be removed. Old iteration's patch that got through, it seems. I'll try to remove it.